bin/control.sh

```diff
@@ -31,24 +31,33 @@ fi
 #
 # Import common functions.
 #
-if [ "${IGNITE_HOME:-}" = "" ];
-    then IGNITE_HOME_TMP="$(dirname "$(cd "$(dirname "$0")"; "pwd")")";
-    else IGNITE_HOME_TMP=${IGNITE_HOME};
+if [ "${IGNITE_HOME:-}" = "" ]; then
```

Comment:
Formatting is not consistent. Let's use:

    if [ CONDITION ]; then
        OPERATOR
    else
        OPERATOR
    fi

Reply:
Made the suggested change.